app/views/support_interface/tasks/index.html.erb

```diff
@@ -65,6 +65,18 @@
     </div>
   </section>
 
+    <section class="app-section app-section--with-top-border">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-heading-m">Generate next cycle test applications</h2>
+        <p class="govuk-body">This task generates mostly-random test applications for the next cycle.</p>
+      </div>
+      <div class="govuk-grid-column-one-third">
+        <%= govuk_button_to 'Generate next cycle test applications', support_interface_run_task_path('generate_next_cycle_test_applications'), secondary: true %>
```

Comment:
Can we make this more explicit which cycle it's referring to, as "next cycle" can easily get confusing, especially when using the switcher?

Suggest we add a reference to the year (eg "2022-2023") and explicitly say "recruitment cycle", as "cycle" also sometimes gets used to describe the academic years of the courses being applied to.

```suggestion
        <h2 class="govuk-heading-m">Generate test applications for the <%= cycle %> recruitment cycle</h2>
        <p class="govuk-body">This task generates 100 test applications for the <%= cycle %> recruitment cycle.</p>
      </div>
      <div class="govuk-grid-column-one-third">
        <%= govuk_button_to "Generate test applications for the #{cycle} recruitment cycle", support_interface_run_task_path('generate_next_cycle_test_applications'), secondary: true %>
```

Reply:
👍